templates/base.html.twig

```diff
@@ -34,6 +35,16 @@
                     {% endfor %}
                 </ul>
             </nav>
+
+            <nav>
```

Comment:
In html roles like `nav`, `main`, ... should only exist once. If there are multiple navigations they need to be labeled with an `aria-label` or `aria-labeledby` tag else accessibility is not given.

Reply:
I can change it, because they are right next to each other. But I think what you are saying is not true. [MDN says multiple `nav` elements can be on a single page](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav). Only [`main` should be unique](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main).

Reply:
Sure from html perspective true but not from accessibility, accessibility tools will fail when having unlabeled multiple navs :)